### ISSUE TITLE
cant able to use my mail id

### ISSUE THREAD

**Issue by manuirapuram123** [HabitRPG/habitica#14173]
[//]: # (Before logging this issue, please contact site administrators from "Report a Bug" in the Habitica website's Help menu. If a GitHub issue is needed, staff will let you know and will most likely log one on your behalf. It is recommended that you don't create a new issue unless advised to.)

[//]: # (Bugs in the mobile apps can be reported via Menu > About > Support.)

[//]: # (If you have a feature request, use "Help > Request a Feature", not GitHub.)

[//]: # (For more guidelines see https://github.com/HabitRPG/habitica/issues/2760)

[//]: # (Fill out relevant information - UUID is found from the Habitica website at User Icon > Settings > API)
### General Info
  * UUID:
  * Browser:
  * OS:

### Description
[//]: # (Describe bug in detail here. Include screenshots if helpful.)

#### Console Errors
[//]: # (Include any JavaScript console errors here.)

**Comment by manuirapuram123** [comment]
I tried to sign up, but when I put in my usual mail id for sign-ups, it told me that the e-mail was already taken. Unfazed, I tried to sign in, figuring that I had made an account and forgotten about it (after all, it's happened before). It then told me that there was no user with that username. Now I'm getting a little worried, so I click "forgot password" and enter my e-mail. That's when it tells me that it's not a registered e-mail. Is anybody else getting these kinds of contradictions? and  i was forced to make an account using my spouse's mail id for reporting this issue
